projects/cashmere/src/lib/pop/directives/menu.directive.ts

```diff
@@ -17,6 +17,14 @@ export class MenuDirective implements AfterContentInit, OnDestroy {
     private unsubscribe$ = new Subject<void>();
 
     ngAfterContentInit() {
+        this.updateSubMenus();
+
+        // Update submenus if they are added dynamically or after check
+        this._subMenus.changes.pipe(takeUntil(this.unsubscribe$)).subscribe(() => this.updateSubMenus());
```

Comment:
thanks again for the catch on this @joeskeen.  Went back to the original AfterContentInit, but added a listener for changes on that query list to update the submenus instead.  Which probably better all around in case the list changes dynamically in the app